app/controllers/catalog_controller.rb

```diff
@@ -504,6 +504,25 @@ def search_tips
     end
   end
 
+  def select_image_area
```

Comment:
This controller is already doing too many things.  Please move this to a new controller that is about "image areas".

Reply:
This is taking more than an hour to figure out. I am getting can can errors, etc.